ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -672,6 +662,58 @@ open class BottomCommandingController: UIViewController, TokenizedControlInterna
 
     private var bottomBarBackgroundView: UIView?
 
+    // MARK: - CommandingItem and CommandingSection filtering
+
+    /// Filters the `heroItems` array for items with that are not hidden
+    private func updateVisibleHeroItems() {
+        let updatedVisibleHeroItems = heroItems.filter { item in
+            return !item.isHidden
+        }
+
+        visibleHeroItems = updatedVisibleHeroItems
+    }
+
+    /// Filters the `expandedListSections` 2D array for items with that are not hidden
+    private func updateVisibleExpandedListSections() {
+        /// Filter for all `CommandingSection`  that have at least 1 visible `CommandingItem`
+        let updatedVisibleExpandedListSections = expandedListSections.filter {$0.items.filter {!$0.isHidden}.count > 0}
+
+        /// Filter all `CommandingItem` that are not hidden and add to a new `CommandingSection` to holds the filtered items
+        visibleExpandedListSections = updatedVisibleExpandedListSections.map({ (section: CommandingSection) -> CommandingSection in
```

Comment:
minor: can `(section: CommandingSection) -> CommandingSection in` just trim down to `section in`?